GliaWidgets/Theme/Theme.swift

```diff
@@ -68,12 +68,10 @@ public class Theme {
     public func applyRemoteConfiguration(_ config: RemoteConfiguration) {
```

Comment:
I'm not sure this method should be `public`, because we will remove possibility to pass Theme to `startEngagement` method. So we will apply remote config internally. 
@dukhovnyi what do you think?